config/merge/merge_test.go

```diff
@@ -1392,3 +1437,113 @@ func TestMerge(t *testing.T) {
 		assert.Equal(t, test.transcript, transcript, "#%d bad transcript", i)
 	}
 }
+
+// We are explicitly testing 3.2.0 because it mistakenly has struct
+// pointers. These should not exist but ended up in the Clevis & Luks
+// structs in spec 3.2.0.
+// https://github.com/coreos/ignition/issues/1132
+func TestMerge_3_2_0(t *testing.T) {
```

Comment:
`TestMergeStructPointers`?  The spec version isn't the relevant part here.  (Same observation for the comment block.)